.github/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 <!-- Use the following format below -->
 <!--  1. [Changed Area] Title of changes - @github username (Name)  -->
 
+## 0.11.0
+
+1. [FMS] TO SPEEDS TOO LOW uses FQI fuel quantity after engine start to avoid spurious warnings - @tracernz (Mike)
```

Comment:
Maybe, although maybe we can skip a 0.10 patch release entirely. 0.10.x stuff can only live in the 0.10 branch so it would need to be cherry-picked to there.

Reply:
As you wish. I've already cherry-picked some commits with some fixes I'd like to have for a 0.10.1. But as we want faster releases anyway, we can postpone it to 0.11 👍